release-doc-in-process/all_vf.TTL

```diff
@@ -178,10 +178,10 @@ vf:recipeOutputOf
 vf:involves
         a                   owl:ObjectProperty ;
         rdfs:label          "involves" ;
-        rdfs:domain         [ owl:unionOf (vf:Commitment vf:Intent) ] ; 
+        rdfs:domain         [ owl:unionOf (vf:Commitment vf:Intent vf:EconomicEvent) ] ; 
         rdfs:range          vf:EconomicResource ;
         vs:term_status      "unstable" ;
-        rdfs:comment        "Economic resource the commitment commits or the intent intends to affect. Could be an actual resource or a classification / specification of resource." .
+        rdfs:comment        "Economic resource referenced by the flow." .
```

Comment:
What do you think about: *Economic resource involved in the flow*

Reply:
I try not to repeat the word, but no problem with me - changed.